packages/sdk/src/cosmwasmclient.ts

```diff
@@ -276,27 +300,34 @@ export class CosmWasmClient {
   public async getCodes(): Promise<readonly Code[]> {
     const result = await this.restClient.listCodeInfo();
     return result.map(
-      (entry): Code => ({
-        id: entry.id,
-        creator: entry.creator,
-        checksum: Encoding.toHex(Encoding.fromHex(entry.data_hash)),
-        source: entry.source || undefined,
-        builder: entry.builder || undefined,
-      }),
+      (entry): Code => {
+        this.anyValidAddress = entry.creator;
+        return {
+          id: entry.id,
+          creator: entry.creator,
+          checksum: Encoding.toHex(Encoding.fromHex(entry.data_hash)),
+          source: entry.source || undefined,
+          builder: entry.builder || undefined,
+        };
+      },
     );
   }
 
   public async getCodeDetails(codeId: number): Promise<CodeDetails> {
-    const getCodeResult = await this.restClient.getCode(codeId);
+    const cached = this.codesCache.get(codeId);
+    if (cached) return cached;
```

Comment:
Style point

I thought linter wanted if body to be inside brackets.

Reply:
Back in the tslint days that was the case. Now we have a more sophisticated `curly` configuration in eslint with the following requirements:
1. any multiline statement must have bracket ("multi-line")
2. if any of the cases in an if/elseif/else constructrion has brackets, all of them need them ("consistent")

This protects especially from this dangerous case:

```js
if (foo)
  bar1();
```

gets refactored to

```js
if (foo)
  bar1();
  bar2();
```

But for many cases where we have a short test and a single expression, we can write it in one line.